src/main/java/com/google/collegeplanner/data/Schedule.java

```diff
@@ -61,6 +64,10 @@ private boolean conflictsWithSchedule(Section section) {
     return sections;
   }
 
+  public void removeLastClass() {
+    sections.remove(sections.size() - 1);
```

Comment:
This will crash if you remove when `sections.size() == 0`. Also, adding a quick test for this would be helpful to ensure this doesn't fail in unexpected ways.